docs/basics/101-138-sharethirdparty.rst

```diff
@@ -0,0 +1,502 @@
+  as necessary.
+
+Thus, from a collaborator's perspective, with the exception of potentially
+installing/setting up the relevant *special-remote*, obtaining your dataset and its
+data is as easy as with any public DataLad dataset.
+While you have to invest some setup effort in the beginning, once this
+is done, the workflows of yours and others are the same that you are already
+very familiar with.
+
+Setting up 3rd party services to host your data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this paragraph you will see how a third party service can be configured
+to host your data. Note that the *exact* procedures are different from service
+to service -- this is inconvenient, but inevitable given the
+differences between the various third party infrastructures.
+The general workflow, however, is the same:
+
+#. Implement the appropriate Git-annex *special-remote* (different
+   from service to service).
+#. Push annexed file content to the third-party service to use it as a storage provider
+#. Share the dataset (repository) via GitHub/GitLab/... for others to install from
+
+If the above steps are implemented, others can :command:`install` or
+:command:`clone` your shared dataset, and :command:`get` or :command:`pull` large
+file content from the remote, third party storage.
+
+.. findoutmore:: What is a special remote
+
+   A special-remote is an extension to Git’s concept of remotes, and can
+   enable :term:`git-annex` to transfer data to and from places that are not Git
+   repositories (e.g., cloud services or external machines such as an HPC
+   system). Don’t envision a special-remote as a physical place or location
+   – a special-remote is just a protocol that defines the underlying transport
+   of your files to and from a specific location.
+
+As an example, let's walk through all necessary steps to publish ``DataLad-101``
+to **Dropbox**. If you instead are interested in learning how to set up a public
+`Amazon S3 bucket <https://aws.amazon.com/s3/?nc1=h_ls>`_, there is a single-page, step-by-step
+walk-through `in the documentation of git-annex <https://git-annex.branchable.com/tips/public_Amazon_S3_remote/>`_
+that shows how you can create an S3 special remote and share data with anyone who
+gets a clone of your dataset, without them needing Amazon AWS credentials. Likewise,
+the documentation provides step-by-step walk-throughs for many other services,
+such as `Google Cloud Storage <https://git-annex.branchable.com/tips/using_Google_Cloud_Storage/>`_,
+`Box.com <https://git-annex.branchable.com/tips/using_box.com_as_a_special_remote/>`__,
+`Amazon Glacier <https://git-annex.branchable.com/tips/using_Amazon_Glacier/>`_,
+`OwnCloud <https://git-annex.branchable.com/tips/owncloudannex/>`__, and many more.
+Here is the complete list: `git-annex.branchable.com/special_remotes/ <https://git-annex.branchable.com/special_remotes/>`_.
+
+For Dropbox, the relevant special-remote to configures is
```

Comment:
Everything that follows would benefit a lot from better support on the datalad end https://github.com/datalad/datalad/issues/1784